lib/services/import_user.rb

```diff
@@ -0,0 +1,36 @@
+class ImportUser
+  EXPECTED_COLUMNS = %w(first_name last_name email birth_date employment_date password)
+  STATUS_NEW  = 'NEW'
+  STATUS_OK   = 'OK'
+  STATUS_FAIL = 'FAIL'
+
+  attr_reader :error
+
+  def initialize(attributes, options = {})
+    @options = { verbose: false }.merge(options)
+    @attributes = attributes.select { |key| EXPECTED_COLUMNS.include?(key) }
+    @status = 'new'
+    @error = ''
+  end
+
+  def insert_to_db
+    User.create!(@attributes)
+    @status = STATUS_OK
+  rescue StandardError => e
```

Comment:
It seems to be not feasible:
```
> rake users:import_from_csv 1
invalid option: 1
```
It can be implemented in the following way:
  - insert user into DB with provided data if user with provided `email` does not exist
  - update user in DB with provided data if user with provided `email` already exists